src/util/misc.js

```diff
@@ -1044,7 +1044,7 @@
      * this is equivalent to remove from that object that transformation, so that
      * added in a space with the removed transform, the object will be the same as before.
      * Removing from an object a transform that scale by 2 is like scaling it by 1/2.
-     * Removing from an object a transfrom that rotate by 30deg is like rotating by 30deg
+     * Removing from an object a transform that rotate by 30deg is like rotating by 30deg
```

Comment:
i think i spelled transform in all possible way, and different people corrected one part of it